src/runtime/dom-particle.ts

```diff
@@ -9,10 +9,13 @@
  */
 
 import {XenStateMixin} from '../../modalities/dom/components/xen/xen-state.js';
-import {DomParticleBase} from './dom-particle-base.js';
+import {DomParticleBase, Model} from './dom-particle-base.js';
 import {Collection, Handle, Variable} from './handle.js';
+import {Runnable} from './functional.js';
 
-interface StatefulDomParticle extends DomParticleBase {
+// TODO(alxr) Where is `ReadonlyMap` coming from? My IDE says that typescript's version should take 1 generic, not 2.
```

Comment:
consider removing comment..

Reply:
I'll def rm the comment. 
I'm curious about where the `ReadonlyMap` type comes from. Shouldn't it be `ReadonlyMap<number>`?

Reply:
Looks like there are two ReadonlyMap definitions if you look in node_modules/typescript/lib

The single arg one appears to be something internal to typescript that uses strings for keys.

The K,V version appears in the lib.es2015.collection.d.ts file, and I believe is offical...

Reply:
Ah, I reinstalled node modules, and now I stop getting the warnings. Thank you.